src/main/java/com/google/devtools/build/lib/remote/common/RemoteCacheClient.java

```diff
@@ -73,7 +73,7 @@ public int hashCode() {
    * @return A Future representing pending download of an action result. If an action result for
    *     {@code actionKey} cannot be found the result of the Future is {@code null}.
    */
-  ListenableFuture<ActionResult> downloadActionResult(
+  FutureCachedActionResult downloadActionResult(
```

Comment:
Can we return `ListenableFuture<CachedActionResult>` instead?

Reply:
done, doing the Futures.transform was new to me ....